test/functional/fixtures/api/es-next/click/test.js

```diff
@@ -64,4 +65,23 @@ describe('[API] t.click()', function () {
                 );
             });
     });
+
+    it('Should not process the same driver status twice', function () {
+        return runTests('./testcafe-fixtures/repeated-driver-status.js', 'Click and wait for page unloading', {
+            shouldFail: true,
+            only:       'chrome'
+        })
+            .catch(function (errs) {
+                errorInEachBrowserContains(errs, 'Button clicked', 0);
+                errorInEachBrowserContains(errs,
+                    '8 |}' +
+                    ' 9 |' +
+                    ' 10 |test(\'Click and wait for page unloading\', async t => {' +
+                    ' 11 |    await t.click(\'#link\');' +
+                    ' 12 |    await wait(1000); >' +
```

Comment:
wron formatting, also there is no need to test whole callsite, just test for the expected line:
```
''> 13 | await t.click('#btn');"
```